src/lib/backend-util.ts

```diff
@@ -0,0 +1,33 @@
+import { writable, get } from 'svelte/store';
+
+const initial = 'https://localhost:5000/';
+const backendUrl = writable<string>(initial);
+
+function setBackendUrl(newUrl: string) {
+    backendUrl.set(newUrl);
+}
+
+function getBackendUrl(): string {
+    return get(backendUrl);
+}
+
+// Backend endpoints
+const plugPower = 'plug/power';
+const configuration = 'configuration';
```

Comment:
Hab nicht in die Swagger geguckt, aber sonst stimmt glaube ich Zeile 24 nicht.
```suggestion
const configuration = 'configuration/';
```